misc/basic_keys.py

```diff
@@ -20,8 +20,8 @@
         "( enter | shock )": "enter",
         "( escape | randall )": "escape",
         "home": "home",
-        "pagedown": "pagedown",
-        "pageup": "pageup",
+        "page down": "pagedown",
```

Comment:
did this need to be changed for you? these work fine for me as is `pageup` and `pagedown`. Maybe we should have `(page up | pageup)` in case they are different for different people

Reply:
Yeah. I needed the fix. I can put in the latter snippet (page up | pageup).

Reply:
Fix this. When with the option in the comment above.